### PR TITLE
Standardise font size when less than 0

### LINKED ISSUE #279: replace font sizes <= 0.0 with default font size
yea, this behaviour should be standardized; it seems like both mac and windows use `12.0` if the passed value is <= 0, and I think we should just enforce that.

_Originally posted by @cmyr in https://github.com/linebender/druid/issues/1156#issuecomment-681165023_

### PR DESCRIPTION
**Pull request by JAicewizard**
fixes  #279

### CODE REVIEW

File: piet/src/util.rs [hunk 1]
```diff
@@ -167,7 +167,7 @@ impl LayoutDefaults {
     pub fn set(&mut self, val: impl Into<TextAttribute>) {
         match val.into() {
             TextAttribute::FontFamily(t) => self.font = t,
-            TextAttribute::FontSize(size) => self.font_size = size,
+            TextAttribute::FontSize(size) => self.font_size = if size <= 0.0 { 12.0 } else { size },
```
Comment (cmyr) [inline piet/src/util.rs:170]: you can use the `DEFAULT_FONT_SIZE` const defined elsewhere in this file. :)

**Review by cmyr — COMMENTED** [review]
Great. We have a `const` for this; otherwise thanks for the fixup. :)

File: piet/src/util.rs (hunk 1 above)
Comment (JAicewizard) [reply piet/src/util.rs:170]: done!

**Review by cmyr — APPROVED** [review]
thanks!

**Pull request merged**